gammapy/image/plotting.py

```diff
@@ -7,34 +7,37 @@
 __all__ = ['colormap_hess', 'colormap_milagro',
            'fits_to_png',
            'GalacticPlaneSurveyPanelPlot',
-           ]
+           'fitsfigure_add_psf_inset',
+           'world2fig',
```

Comment:
I don't think `world2fig` needs to be part of the public gammapy API in `gammapy.image`, no?
If you want to expose it, please choose a longer, more descriptive name.